test/specs/auraLsp.e2e.ts

```diff
@@ -5,29 +5,104 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 import { step } from 'mocha-steps';
+import { TextEditor } from 'wdio-vscode-service';
 import { TestSetup } from '../testSetup';
+import * as utilities from '../utilities';
 
 describe('Aura LSP', async () => {
   let testSetup: TestSetup;
   let projectName: string;
 
   step('Set up the testing environment', async () => {
+    utilities.log(`AuraLsp - Set up the testing environment`);
     testSetup = new TestSetup('AuraLsp', false);
     await testSetup.setUp();
     projectName = testSetup.tempProjectName.toUpperCase();
+
+    // Create Aura Component
+    await utilities.createAura('aura1');
   });
 
-  step('what?', async () => {
-    // TODO: implement
-    expect(1).toBe(1);
+  step('Verify Extension is Running', async () => {
+    utilities.log(
+      `${testSetup.testSuiteSuffixName} - Verify Extension is Running`
+    );
+    // Using the Command palette, run Developer: Show Running Extensions
+    const workbench = await browser.getWorkbench();
+    await utilities.runCommandFromCommandPrompt(
+      workbench,
+      'Developer: Show Running Extensions',
+      1
+    );
+
+    utilities.log(
```

Comment:
We can probably skip this step to enable the LWC extension, since we don't need it to test aura